src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -87,7 +87,7 @@ public Xcode(string target, string arch)
         var entitlements = new List<KeyValuePair<string, string>>();
 
         bool hardenedRuntime = false;
-        if (Target == TargetNames.MacCatalyst && !(forceInterpreter || forceAOT)) {
+        if (Target == TargetNames.MacCatalyst && !forceAOT) {
             hardenedRuntime = true;
 
             /* for mmmap MAP_JIT */
```

Comment:
Can you leave a `/* FIXME: figure out why we are trying to JIT trampolines, disable it, and remove the JIT entitlement when we're using the interpreter */` just before the `entitlements.Add` for the JIT